ci/jenkins/jobs/macros.yaml

```diff
@@ -112,17 +112,29 @@
       - shell: |-
           #!/bin/bash
           set -ex
+          cat "/var/lib/jenkins/host-local.json" | CNI_COMMAND=ADD CNI_CONTAINERID="$BUILD_TAG" CNI_NETNS=/dev/null CNI_IFNAME=dummy0 CNI_PATH=. /usr/bin/host-local > ip-result.json
```

Comment:
Better to allocate and release IP in test-vmc.sh.

It would be like:
```
#!/bin/bash
set -e

function release_static_ip {
  cat "/var/lib/jenkins/host-local.json" | CNI_COMMAND=DEL CNI_CONTAINERID="$BUILD_TAG" CNI_NETNS=/dev/null CNI_IFNAME=dummy0 CNI_PATH=. /usr/bin/host-local
  echo released IP
}
trap release_static_ip Exit

cat "/var/lib/jenkins/host-local.json" | CNI_COMMAND=ADD CNI_CONTAINERID="$BUILD_TAG" CNI_NETNS=/dev/null CNI_IFNAME=dummy0 CNI_PATH=. /usr/bin/host-local > ip-result.json
CONTROL_PLANE_IP=$(cat ip-result.json | jq -r '.ips[0].address' | awk -F '/' '{print $1}')
echo Allocated "$CONTROL_PLANE_IP"
```

Reply:
Updated